addon/utils/dom-helpers.ts

```diff
@@ -1,6 +1,7 @@
 import { A } from '@ember/array';
 import { PernetSelection, PernetSelectionBlock } from '@lblod/ember-rdfa-editor/editor/pernet'
 import { RichNode } from '@lblod/ember-rdfa-editor/editor/raw-editor'
+import { isEmptyRdfaAttributes } from './rdfa/rdfa-rich-node-helpers';
```

Comment:
was this import missing or where is it coming from?

Reply:
I have no idea, probably an accidental auto-import